### PR TITLE
bugs fix

### PR DESCRIPTION
**Pull request by SRINI2410**
Fixed bugs in K8s deployment

### CODE REVIEW

File: K8s-deployment/Charts/databroker/values.yaml [hunk 1]
```diff
@@ -797,7 +797,7 @@ livenessProbe:
 ##
 readinessProbe:
   enabled: true
-  initialDelaySeconds: 10
+  initialDelaySeconds: 120
```
Comment (abhi4578) [inline K8s-deployment/Charts/databroker/values.yaml:800]: Please document the reason for increasing the initialDelaySeconds.

File: K8s-deployment/K8s-cluster/addons/ingress-controller/memcached/example-resource-values.yaml [hunk 2]
```diff
@@ -1,11 +1,11 @@
 resources:
   limits:
-    cpu: 200m
-    memory: 256Mi
+    cpu: 50m
+    memory: 0.1Mi
```
Comment (abhi4578) [inline K8s-deployment/K8s-cluster/addons/ingress-controller/memcached/example-resource-values.yaml:4]: Hmm, the resource request and limit seems very low, please make it as before

**Review by abhi4578 — CHANGES_REQUESTED** [review]

**Comment by abhi4578** [comment]
Also can all API server image value in values.yaml be changed to latest 5.5.0 release image tags @SRINI2410

File: K8s-deployment/K8s-cluster/addons/ingress-controller/ingress-nginx/nginx-values.yaml [hunk 3]
```diff
@@ -291,23 +291,23 @@ controller:
...
-    #       - key: app.kubernetes.io/instance
-    #         operator: In
-    #         values:
-    #         - ingress-nginx
-    #       - key: app.kubernetes.io/component
-    #         operator: In
-    #         values:
-    #         - controller
-    #     topologyKey: "kubernetes.io/hostname"
+    podAntiAffinity:
```
[the remark this answers is not in the document]
Comment (SRINI2410) [reply K8s-deployment/K8s-cluster/addons/ingress-controller/ingress-nginx/nginx-values.yaml:294]: There was a mismatch of label (app.kubernetes.io/instance: nginx) used by official NGINX helm chart and the custom chart developed ( app.kubernetes.io/instance: ingress-nginx).

File: K8s-deployment/Charts/databroker/values.yaml (hunk 1 above)
Comment (SRINI2410) [reply K8s-deployment/Charts/databroker/values.yaml:800]: Probe to check readiness was taking more time than 10s. Hence, change value to some arbitrary value based on trial and error.

File: K8s-deployment/K8s-cluster/addons/ingress-controller/memcached/example-resource-values.yaml (hunk 2 above)
Comment (SRINI2410) [reply K8s-deployment/K8s-cluster/addons/ingress-controller/memcached/example-resource-values.yaml:4]: Sure, will do necessary changes.

**Comment by SRINI2410** [comment]
API servers images tag has been updated @abhi4578

File: K8s-deployment/Charts/resource-server/values.yaml [hunk 4]
```diff
@@ -72,7 +72,7 @@ diagnosticMode:
 image:
   registry: ghcr.io
   repository: datakaveri/rs-depl
-  tag: 5.5.0-alpha-78ba6a3
+  tag: 5.5.0-alpha-5249e09
```
Comment (abhi4578) [inline K8s-deployment/Charts/resource-server/values.yaml:75]: Add latest 5.5.0 images without alpha prefix

Comment (SRINI2410) [reply K8s-deployment/Charts/resource-server/values.yaml]: Same has been updated @abhi4578

**Review by abhi4578 — APPROVED** [review]

**Pull request merged**